src/components/layout/layout.tsx

```diff
@@ -85,6 +86,7 @@ const Layout = ({
     is_ppc_redirect = false,
     margin_top = '',
     type = 'default',
+    show_footer = true,
```

Comment:
I think it's true by default so might not need **=true**. Could you check?

Reply:
By default it will be `undefined` 
So system will recognise this as false and it will get hide from all the pages where we are not passing any prop.